### PR TITLE
Change to about text, I agree and cookies text

### PR DESCRIPTION
**Pull request by danieltwigt**
Change to about text, I agree and cookies text, for CHASM, legal feedback, naming and some further info.

### CODE REVIEW

File: assets/docs/about-text.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
-The Deltares Global Data Services platform has been developed by Deltares as a free, web-based application to support the study and sharing of integrated water and subsoil-related data. The platform provides indicative data, that is primarily intended for professional specialists and researchers.
+This is a beta release of the Blue Earth Data platform. The Blue Earth Data platform has been developed by Deltares as a free, web-based application to support the study and sharing of integrated water and subsoil-related data. The platform provides indicative data, which is primarily intended for professional specialists and researchers.  
+&nbsp;
```
Comment (vvandijck) [inline assets/docs/about-text.md:2]: Are all these no-break spaces necessary? If it's just to create extra whitespace we can solve this by changing the styling in CSS.

Comment (danieltwigt) [reply assets/docs/about-text.md:2]: If we can solve this by changing the styling in the CSS that would be fine as well. I just modified the text. Most of the no-break spaces were already in there. So maybe also check with Cindy. Also note that if changes to the CSS are required, this is probably up to you or Cindy (to do it well).

Comment (vvandijck) [reply assets/docs/about-text.md:2]: Ok, then we will increase spacing in the styling.

Comment (danieltwigt) [reply assets/docs/about-text.md:2]: Should I commit modified version of these files without the no-break spaces in this case? Or will you also modify this?

Comment (vvandijck) [reply assets/docs/about-text.md:2]: If it's easy, yes please!

Comment (danieltwigt) [reply assets/docs/about-text.md:2]: Nog ever ter bevestiging: geldt dit voor zowel user-agreements.md, cookie-agreements.md en about-text.md? En kunnen alle no-break spaces uit deze files verwijderd worden?

Comment (vvandijck) [reply assets/docs/about-text.md:2]: Yes, de styling geldt voor alle markdown teksten

Comment (danieltwigt) [reply assets/docs/about-text.md]: Ik heb de no-break spaces uit de bestanden verwijderd.

**Pull request merged**